src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public class Ball {
+    public static void ball(String color, int number) {
+
+    }
+
+    @Override
+    public String toString() {
+        return super.toString();
```

Comment:
There's no point in overriding and leaving the default implementation, write your own, meaningful one